glassfish-on-suse/azuredeploy.parameters.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
When a parameter has a defaultValue declared in the template (e.g. _artifactsLocation, vmsize, osversion - they aren't needed here...